weather_mv/README.md

```diff
@@ -48,6 +45,7 @@ _Usage Examples_:
 ```bash
 weather-mv --uris "gs://your-bucket/*.nc" \
            --output_table $PROJECT.$DATASET_ID.$TABLE_ID \
+           --temp_location "gs://$BUCKET/tmp" \  # Needed for batch writes to BigQuery
```

Comment:
Is temp_location always required? It should be added above to 'Common options' and marked as '(required)' if applicable.

Reply:
Thanks! This argument is not always required – only for batch uploads (the BigQuery transform needs it). For the streaming use case, this is not necessary.